CONTRIBUTING.md

```diff
@@ -13,6 +15,21 @@ We lovingly stole these guidelines from [Google's Fuchsia project](https://fuchs
 
 * [Documentation Style](#documentation-style)
 
+## PR Checklist
+
+- [ ] Is the audience and objective of the document clear from the first paragraph? E.g. a document
```

Comment:
I realized after I wrote this that it could also be clear just from its location on the website. Not everything should have to say "for a developer" if it's in the "Developer Docs" section. Audience/objective should be clear from context.
```suggestion
- [ ] Are the audience and objective of the document clear? E.g. a document
```